cypress/e2e/spotlight/spotlight.spec.ts

```diff
@@ -296,7 +297,7 @@ describe("Spotlight", () => {
             cy.spotlightResults().eq(0).should("contain", room3Id);
         }));
     });
-    */
+    *//*
```

Comment:
commenting out tests is not great, you can use `it.skip` but also disabling tests should be done in its own PR with a good reason.

Reply:
Oh f... thanks for catching that, that wasn’t even intended to be committed at all. I wish cypress-UI would allow running individual tests from the UI

Reply:
@justjanne doesn't solve not having a UI but you can use `it.only` to run just a single test within a spec rather than skipping/commenting out the rest